extras/module-node/duk_module_node.c

```diff
@@ -182,6 +182,12 @@ static void duk__push_module_object(duk_context *ctx, const char *id) {
 
 	/* module.require */
 	duk__push_require_function(ctx, id);
+	if (main) {
+		duk_dup(ctx, -2);
+	} else {
+		duk_push_undefined(ctx);
```

Comment:
Wait, so doesn't this mean `require.main` is only set in the main module?  That doesn't seem right.